os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -838,55 +871,81 @@ _timerHandle _tizenrt_timerCreate(const signed char *pcTimerName, osdepTickType
 {
 	struct timer_list_priv *timer = (struct timer_list_priv *)_tizenrt_zmalloc(sizeof(struct timer_list_priv));
 	if (timer == NULL) {
-		DBG_ERR("Fail to alloc priv");
+		DBG_ERR("Fail to alloc priv\n");
 		return NULL;
 	}
 	timer->work_hdl = (struct work_s *)_tizenrt_zmalloc(sizeof(struct work_s));
 	if (timer->work_hdl == NULL) {
-		DBG_ERR("Fail to alloc timer->work_hdl");
+		DBG_ERR("Fail to alloc timer->work_hdl\n");
 		kmm_free(timer);
 		return NULL;
 	}
 	timer->live = 0;
 	timer->timevalue = xTimerPeriodInTicks;
 	timer->data = pvTimerID;
 	timer->function = pxCallbackFunction;
+
+	if(_tizenrt_timer_mutex == NULL) {
+		unsigned int irq_flags = save_and_cli();
+		if(_tizenrt_timer_mutex == NULL) {
+			_tizenrt_mutex_init(&_tizenrt_timer_mutex);
+			INIT_LIST_HEAD(&_tizenrt_timer_table);
+		}
+		restore_flags(irq_flags);
+	}
+
+	struct _tizenrt_timer_entry *timer_entry;
+	timer_entry = _tizenrt_zmalloc(sizeof(struct _tizenrt_timer_entry));
+	if (timer_entry == NULL) {
+		kmm_free(timer->work_hdl);
+		kmm_free(timer);
+		return NULL;
+	}
+	memset(timer_entry, 0, sizeof(struct _tizenrt_timer_entry));
```

Comment:
This is duplicated zero initialization for zmalloc and here. Let's remove this line or use malloc instead of zmalloc.

Reply:
Done modify, thanks